scripts/release/asset_publish.py

```diff
@@ -406,6 +406,17 @@ def create_asset(
         failure_list.append(asset)
 
 
+def stringify_dictionary(dictionary):
+    """Convert the type of values to string."""
+    for name, value in dictionary.items():
```

Comment:
what about just `json.dumps(dictionary)`?

Reply:
I see, so basically we have moved the method up and we are also saving value as string. Ignore this one as well